nixos/modules/services/x11/desktop-managers/gnome.nix

```diff
@@ -215,7 +221,9 @@ in
       services.gnome.core-shell.enable = true;
       services.gnome.core-utilities.enable = mkDefault true;
 
-      services.displayManager.sessionPackages = [ pkgs.gnome-session.sessions ];
+      services.displayManager.sessionPackages = [
+        (pkgs.gnome-session.override { x11Support = !cfg.waylandOnly; }).sessions
```

Comment:
Also not sure if inclusion in a module is enough for Hydra to put it into cache. But maybe NixOS tests are enough?

Reply:
We could enable the option [here](https://github.com/NixOS/nixpkgs/blob/a466f5f18289cb29f58f0459f362132b34af254b/nixos) in the release jobset. Given that we are technically using both packages as you explained above, this should ensure both are always cached